data/publiccloud/azure_runcmd.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash -eu
+set -o pipefail
+# Create a vm and use run-command to execute RunShellscript
+# list, show , delete the created resource
+
+#Required AZ setting variable block
+grpname="${1:-oqaclitest}"
+location="${2:-westus}"
+vmname="${3:-oqacliruncmdvm}"
+ssh_key="${4:-oqaclitest-sshkey}"
+vmximagename="${5:-UbuntuLTS}"
+
+#local variable block
+account="Azure RD"
+admin="azureuser"
+runcmd="oqaclirc"
+cli_cnt=0
+total_cnt=16
+fail_cnt=0
+cil_list=''
+
+echo "**Start of AZURE CLI run-command test"
+echo "*************************************"
+
+# Resource creation
+az account set -n "$account"
+echo "Creating resource group"
+let "cli_cnt+=1"
+cli_list='az account set\n'
+
+az group create -n $grpname -l $location -o table
+echo "Created group $grpname"
+let "cli_cnt+=1"
+cli_list="$cli_list az group create\n"
+
+#start vm provision
+az vm create \
+    --name $vmname \
+    --resource-group $grpname \
+    --location $location \
+    --public-ip-sku Standard \
+    --image $vmximagename \
+    --generate-ssh-keys \
+    --admin-username $admin \
+    --enable-agent true \
+    --enable-auto-update true \
+    --size Standard_D2s_v3
+echo "vm created"
+let "cli_cnt+=1"
+cli_list="$cli_list az vm create\n"
+
+echo "Invoke run-command to change AutoUpdate.Enabled=y and restart waagent.service"
+az vm run-command invoke \
```

Comment:
this single call is enough to make sure that `az vm run-command invoke` is working , please remove all below let's not overcomplicate this

Reply:
we can remove the rest